sql/2019/17_CDN/17_18.sql

```diff
@@ -0,0 +1,20 @@
+#standardSQL
+# 17_18: Percentage of responses that were pushed
+SELECT
+  client,
+  cdn,
+  COUNTIF(pushed) AS freq,
+  COUNT(0) AS total,
+  ROUND(COUNTIF(pushed) * 100 / COUNT(0), 2) AS pct
+FROM (
+  SELECT
+    client,
+    _cdn_provider AS cdn,
+    JSON_EXTRACT(payload, '$._was_pushed') IS NOT NULL AS pushed
+  FROM
+    `httparchive.almanac.requests`)
+GROUP BY
```

Comment:
need to exclude htmlpages since it doesn't make sense to push the page you are requesting.